scripts/clang-format-style-file

```diff
@@ -82,13 +82,17 @@ IncludeCategories:
     Priority:     -5,
   }
 - {
-    Regex:        '^"boost/',
+    Regex:        '^"Qt[A-Za-z]+/Q[A-Za-z]+"$',
```

Comment:
```suggestion
    Regex:        '^"Qt[^/]+/',
```

Reply:
I wanted to be extra explicit to minimize possibility of false positives. Those filenames shouldn't use anything but letters if I understand correctly.

Reply:
`root/include/QtQuick3DAssetImport`

Reply:
I'm stupid, sorry :)

Reply:
Done.

Reply:
```
find root/include/Qt* -type f -not -name "Q*"
```
I think just matching `Qt*/` is OK.

Reply:
`IncludeCategories` regex is case-insensitive by default, that leaves us with `QtBodymovin/bmglobal.h`, `QtZlib/zconf.h` and `QtZlib/zlib.h` which are not meant to be included explicitly.

But ok, I'll remove the Q just in case someone trying to include `zlib` in this weird fashion.